spec/features/cart/remove_item_spec.rb

```diff
@@ -41,6 +41,54 @@
         expect(page).to have_css("#cart-item-#{@pencil.id}")
         expect(page).to have_css("#cart-item-#{@paper.id}")
       end
+      it 'can render the button to increment a decrement'do
+        visit "/items/#{@tire.id}"
+
+        click_on "Add To Cart"
+
+        visit "/cart"
+
+        within "#cart-item-#{@tire.id}" do
+          expect(page).to have_button("+")
+          expect(page).to have_button("-")
+        end
+
+      end
+
+
+    end
+      it 'can press the buttons and add more items and decrement passt 0'do
+      meg = Merchant.create(name: "Meg's Bike Shop", address: '123 Bike Rd.', city: 'Denver', state: 'CO', zip: 80203)
+
+      tire = meg.items.create(name: "Gatorskins", description: "They'll never pop!", price: 100, image: "https://www.rei.com/media/4e1f5b05-27ef-4267-bb9a-14e35935f218?size=784x588", inventory: 12)
+      pencil = meg.items.create(name: "Pencil", description: "They'll never pop!", price: 100, image: "https://images-na.ssl-images-amazon.com/images/I/31BlVr01izL._SX425_.jpg", inventory: 2)
+
+      visit "/items/#{tire.id}"
+
+      click_on "Add To Cart"
+
+      visit "/items/#{tire.id}"
+
+      click_on "Add To Cart"
+
+      visit "/items/#{pencil.id}"
+
+      click_on "Add To Cart"
+
+      visit "/cart"
+
+
+      within "#cart-item-#{tire.id}" do
+        click_on "-"
+        click_on "-"
+      end
+      expect(page).to_not have_css("#cart-item-#{tire.id}")
+      expect(page).to have_css("#cart-item-#{pencil.id}")
+
+      within "#cart-item-#{pencil.id}" do
+        click_on "+"
+        expect(page).to_not have_button("+")
+      end
     end
```

Comment:
I like that you tested the button '+' disappears once the item's quantity has been reached.